config/locales/en.yml

```diff
@@ -266,8 +267,11 @@ en:
 
           When you are ready to upgrade your Rancher instance, please see the [Rancher docs](https://docs.ranchermanager.rancher.io/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades)
           for details.
-        failed_caption: "**Cleaning up the resources failed. Something went wrong.**"
-        failed_info: "The resources created should be cleaned up manually, if any."
+        failed_caption: "**Deployment failed. Something went wrong during the deployment.**"
+        failed_info: "The resources created should be cleaned up, if any. Retry another deployment once the issue has been resolved."
+        failed_wrapup_caption: "**Cleaning up the resources failed. Something went wrong: %{error}**"
```

Comment:
IMO `failed_wrapup_caption` is ambiguous. Consider `failed_cleanup_caption`.

Reply:
Fixed